### ISSUE TITLE
Remove default placeholder 'Enter a new tag'

### ISSUE THREAD

**Issue by rspandi1985** [Gbuomprisco/ngx-chips#828]
Is it possible to remove default placeholder or change it? I know after adding one item it will show changed placeholder. But can we show new placeholder text as soon as we load tag input?

**Comment by Gbuomprisco** [comment]
just pass the string you want to the inputs

**Comment by rspandi1985** [comment]
That is not working. it still shows enter a new tag on initial load. Only after adding one item it will show your placeholder text. Can you please check?

**Comment by andre-f-paggi** [comment]
Having the same issue here, suggestions?

**Comment by andre-f-paggi** [comment]
> Having the same issue here, suggestions?

Sorry, found the answer in another issue:
https://github.com/Gbuomprisco/ngx-chips/issues/829#issuecomment-427553908
